### PR TITLE
fix: install app dropdown is too large

### LINKED ISSUE #11561: [CAL-2532] install app dropdown way too large
should only be as wide as the content

https://github.com/calcom/cal.com/assets/8019099/6342e962-d22e-426b-919f-a79b4a4acc33

CleanShot 2023-09-27 at 22 38 48@2x

<sub>[CAL-2532](https://linear.app/calcom/issue/CAL-2532/install-app-dropdown-way-too-large)</sub>

### PR DESCRIPTION
**Pull request by bhargavagonugunta**
## What does this PR do?
Fixes #11561 

Screenshot 2023-09-28 at 3 46 35 AM

### CODE REVIEW

**Review by Udit-takkar — COMMENTED** [review]
Screenshot 2023-09-28 at 4 07 12 PM

I don't see this bug on latest main branch

File: apps/web/components/apps/InstallAppButtonChild.tsx [hunk 1]
```diff
@@ -76,6 +76,7 @@ export const InstallAppButtonChild = ({
       </DropdownMenuTrigger>
       <DropdownMenuPortal>
         <DropdownMenuContent
+          className="w-[45vh]"
```
Comment (PeerRich) [inline apps/web/components/apps/InstallAppButtonChild.tsx:79]: this will cause issues if the name is too long

Comment (PeerRich) [inline apps/web/components/apps/InstallAppButtonChild.tsx:79]:
```suggestion
          className="max-w-min"
```

**Review by PeerRich — APPROVED** [review]

**Comment by PeerRich** [comment]
fixed. using className="max-w-min" instead

**Review by PeerRich — APPROVED** [review]

**Pull request merged**